Logger/FileLogger.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.IO;
+namespace Logger;
+
+public class FileLogger : BaseLogger
+{
+    private string FilePath { get; set; }
```

Comment:
```suggestion
    public string FilePath { get; private set; }
```
You could probably do something like this if you want the FilePath to be publically accessible (even though it's sorta against what the Assignment says), that way you can get rid of the `GetFilePath` on line `39`.